.circleci/config.yml

```diff
@@ -18,24 +20,15 @@ jobs:
             pip install sphinx-gallery
             pip install .
             cd doc
-            make html
+            make doctest
```

Comment:
The recommended way of running doctests is with `pytest --doctest-modules`, not via the sphinx makefile.